copper-server-impl/src/main/java/com/pogeyan/cmis/impl/services/CmisDiscoveryService.java

```diff
@@ -45,20 +45,25 @@
 import org.slf4j.LoggerFactory;
 
 import com.pogeyan.cmis.api.auth.IUserObject;
+import com.pogeyan.cmis.api.data.IBaseObject;
+import com.pogeyan.cmis.api.data.ISpan;
 import com.pogeyan.cmis.api.data.common.TokenChangeType;
 import com.pogeyan.cmis.api.data.services.MDiscoveryServiceDAO;
 import com.pogeyan.cmis.api.data.services.MTypeManagerDAO;
 import com.pogeyan.cmis.api.utils.Helpers;
 import com.pogeyan.cmis.impl.factory.DatabaseServiceFactory;
-import com.pogeyan.cmis.api.data.IBaseObject;
+import com.pogeyan.cmis.tracing.TracingApiServiceFactory;
 
 public class CmisDiscoveryService {
 	private static final Logger LOG = LoggerFactory.getLogger(CmisDiscoveryService.class);
 
 	public static class Impl {
-		public static ObjectList getContentChanges(String repositoryId, Holder<String> changeLogToken,
-				Boolean includeProperties, String filter, String orderBy, Boolean includePolicyIds, Boolean includeAcl,
-				BigInteger maxItems, ObjectInfoHandler objectInfos, IUserObject userObject) {
+		public static ObjectList getContentChanges(String baseMessageId, String repositoryId,
```

Comment:
Can we add "baseMessageId" to the end of the method? and please don't format this code.